Gemfile

```diff
@@ -2,7 +2,7 @@ require './prox_gem'
 
 prox_gem 'clock'
 prox_gem 'attribute'
-prox_gem 'null_attr'
+prox_gem 'subst_attr'
```

Comment:
These prox gems are needed because of telemetry logger, per our discussion. I will go ahead and merge the pull request.